src/main/asciidoc/transactions.adoc

```diff
@@ -33,82 +41,257 @@ static class Config extends AbstractCouchbaseConfiguration {
+    }
 }
 ----
 ====
 
-Once the `@Bean` is configured, you can autowire it from your service (or any other class) to make use of it. Please
-see the https://docs.couchbase.com/java-sdk/3.0/howtos/distributed-acid-transactions-from-the-sdk.html[Reference Documentation]
-on how to use the `Transactions` class. Since you need access to the current `Collection` as well, we recommend you to also
-autowire the `CouchbaseClientFactory` and access it from there:
+Please see the https://docs.couchbase.com/java-sdk/current/howtos/distributed-acid-transactions-from-the-sdk.html[Reference Documentation]
+
+== Transactions with TransactionTemplate
+The TransactionTemplate class simplifies programmatic transaction demarcation and transaction exception handling.
+The central method is execute, supporting transactional code that implements the TransactionCallback interface. This template handles the transaction lifecycle and possible exceptions such that neither the TransactionCallback implementation nor the calling code needs to explicitly handle transactions.
+Typical usage: Allows for writing low-level data access objects that use resources such as JDBC DataSources but are not transaction-aware themselves. Instead, they can implicitly participate in transactions handled by higher-level application services utilizing this class, making calls to the low-level services via an inner-class callback object.
+Can be used within a service implementation via direct instantiation with a transaction manager reference, or get prepared in an application context and passed to services as bean reference. Note: The transaction manager should always be configured as bean in the application context: in the first case given to the service directly, in the second case given to the prepared template.
+Supports setting the propagation behavior and the isolation level by name, for convenient configuration in context definitions.
+
+.Using the TransactionTemplate
+====
+[source,java]
+----
+// 5.0.0-M5 transaction template bean does not exist, the TransactionTemplate must be explicitly created.
+@Autowired TransactionManager transactionManager;
+TransactionTemplate transactionTemplate = new TransactionTemplate(transactionManager);
+
+// After M6 transaction template bean exists
+@Autowired TransactionTemplate transactionTemplate;
+
+    transactionTemplate.executeWithoutResult(status -> {
+        Person p = personOperations.findById(Person.class).one(id);
+        p.setFirstname(p.getFirstname()+" Junior");
+        personOperations.replaceById(Person.class).one(p);
+    });
+----
+====
+
+== Transactions with CouchbaseTransactionManager
+
+Note: Reactive Couchbase Transactions cannot be supported via the 
+@Transactional annotation. See the section "Transactions with CouchbaseTransactionalOperator"
+for supporting Reactive Couchbase Transactions using the Transactional Operator.
+ 
+@Transactional Describes a transaction attribute on an individual method or on a class.
```

Comment:
It's a poor choice of words (especially since there is a TransactionAttribute object!). Better would be "@Transactional defines the method (or methods of the class) as executing in a Couchbase Transaction (by means of a method interceptor that leverages a Couchbase Transaction Manager).

Reply:
For my reference, this is now resolved.